components/match2/wikis/ageofempires/match_summary.lua

```diff
@@ -46,28 +47,22 @@ function CustomMatchSummary._determineWidth(match)
 end
 
 ---@param match MatchGroupUtilMatch
----@return MatchSummaryBody
+---@return Widget
 function CustomMatchSummary.createBody(match)
-	local body = MatchSummary.Body()
-
-	if match.dateIsExact or (match.timestamp ~= DateExt.defaultTimestamp) then
-		body:addRow(MatchSummary.Row():addElement(
-			DisplayHelper.MatchCountdownBlock(match)
-		))
-	end
-
-	Array.forEach(match.games, function(game)
-		body:addRow(CustomMatchSummary._createGame(game, {
+	local showCountdown = match.timestamp ~= DateExt.defaultTimestamp
```

Comment:
is `dateIsExact` not relevant anymore?

Reply:
Every game that has an exact date, will also have a timestamp that is not the default timestamp (as that is in year 1000), so the check becomes redundant

Reply:
not true in the following cases:
- manual input without timezone
- dateFallBack wiki vars in play (only on some wikis the case)

Reply:
The previous logic was `dateIsExact OR timestamp is not defaultTimestamp`. Any case where dateIsExact is true, timestamp cannot be defaultTimestamp, as video games weren't invented at defaultTimestamp

Reply:
yeah true

Reply:
Ah, I misread and thought it was an AND.

Would it make sense to move that check entirely into the countdown widget?

Reply:
Yeah but there's no widget for it yet